packages/modules-sdk/src/medusa-app.ts

```diff
@@ -112,6 +119,23 @@ async function initializeLinks(linkModules, injectedDependencies) {
   }
 }
 
+async function getMigrationsFromLinkModule(
+  linkModule: string
+): Promise<RunMigrationFn | undefined> {
+  try {
+    const { runMigrations } = await import(linkModule)
+
+    return runMigrations
+  } catch (err) {
+    console.warn(
+      `Error importing runMigrations from LinkModule - ${linkModule}`,
+      err
+    )
+
+    return undefined
```

Comment:
suggestion: If we can't import the migration function the link won't work and we allow the app to start. Should we instead throw and prevent the app to start in a non working state?